.changes/unreleased/Fixes-20231021-154237.yaml

```
@@ -0,0 +1,6 @@
+kind: Fixes
+body: Patch for json inline --show
+time: 2023-10-21T15:42:37.406853-06:00
+custom:
+  Author: matt.winkler@dbtlabs.com
```

@matt-winkler any reason not to have the author be your GitHub handle?

I think GitHub handles are necessary for the hyperlinks within the [changelog](https://github.com/dbt-labs/dbt-bigquery/blob/1.6.latest/CHANGELOG.md) to work correctly.

```suggestion
  Author: matt-winkler
```

---

nope!